src/typing_extensions.py

```diff
@@ -3020,6 +3024,17 @@ def _check_generic(cls, parameters, elen=_marker):
 
         This gives a nice error message in case of count mismatch.
         """
+        # If substituting a single ParamSpec with multiple arguments
+        # we do not check the count
+        if (inspect.isclass(cls) and issubclass(cls, typing.Generic)
+            and len(cls.__parameters__) == 1
+            and isinstance(cls.__parameters__[0], ParamSpec)
```

Comment:
Do we need to check both typing.ParamSpec and typing_extensions.ParamSpec?

Reply:
I think it might be useful to have a test on 3.10+ that uses `typing.Concatenate` and `typing.ParamSpec`, which are different from the `typing_extensions` version.

Reply:
Tests added & like noted in #489 for Concatenate a patch was necessary.